### ISSUE TITLE
TypeError: __init__() got an unexpected keyword argument 'x_watson_learning_opt_out

### ISSUE THREAD

**Issue by kgarnick** [watson-developer-cloud/python-sdk#430]
Thanks for your work on this library. I'm stuck getting speech diarization to run, as I get
TypeError: __init__() got an unexpected keyword argument 'x_watson_learning_opt_out'
no matter what input wav file I use. I've double checked my credentials as well, which doesn't seem to be the source of the issue. Any insight is much appreciated!
```py
import json, sys
from os.path import join, dirname
from watson_developer_cloud import SpeechToTextV1

wav_file = sys.argv[1]
output_json = wav_file.replace('.wav', '.json')

speech_to_text = SpeechToTextV1(
    username='USERNAME',
    password='PASSWORD',
    x_watson_learning_opt_out=True
)

print(json.dumps(speech_to_text.models(), indent=2))

print(json.dumps(speech_to_text.get_model('en-US_BroadbandModel'), indent=2))

with open(wav_file, 'rb') as audio:
       result = speech_to_text.recognize(audio, content_type='audio/wav', timestamps=True,
           word_confidence=True, speaker_labels=True)
       with open(output_json, 'w') as f:
           json.dump(result, f, ensure_ascii=False)
       print(result)
```

**Comment by ehdsouza** [comment]
Hi @kgarnick ,
We have listed in the migration guide: https://github.com/watson-developer-cloud/python-sdk/wiki/Migration that the init method no longer takes in `x_watson_learning_opt_out`.

```python
speech_to_text = SpeechToTextV1(
    username='USERNAME',
    password='PASSWORD',
)
```

**Comment by ammardodin** [comment]
The mechanism for setting the x-watson-learning-opt-out header has changed in the latest version of the SDK. What you need to do now is set it with the default_headers method after creating the service, like so:

```python
speech_to_text = SpeechToTextV1(username = user_name, password = password, version = version)
speech_to_text.set_default_headers({'x-watson-learning-opt-out': "true"})
```

**Comment by kgarnick** [comment]
Hi @ehdsouza and @ammardodin,

Thanks for the help! Unfortunately, removing that argument gets me another error:

`TypeError: Object of type 'method' is not JSON serializable`

Thanks again!

**Comment by ehdsouza** [comment]
@kgarnick,

`models()` is a deprecated function, use `list_models()`
```python
print(json.dumps(speech_to_text.list_models(), indent=2))
```

**Comment by ehdsouza** [comment]
This should be fixed in the new release v1.3.0
